spec/README.mediawiki

```diff
@@ -311,9 +311,7 @@ respond with its current configuration.
 |-
 | swapconf || int || minimum confirmations for swap transactions
 |-
-| fundconf || int || minimum confirmations for backing UTXOs
-|-
-| scripts  || &#91;string&#93; || acceptable backing UTXO script types. minimum &#91;"P2PKH"&#93;
```

Comment:
I wonder if we'll want some "extra data" field to capture asset-specific coin data constraints.